### PR TITLE
Changes to AWS Removing Extension nodes chapter

### PR DESCRIPTION
**Pull request by ianf77**
Preliminary PR

### CODE REVIEW

**Review by zkayyali812 — APPROVED** [review]

File: stories/assembly-aap-aws-removing-extension-nodes.adoc [hunk 1]
```diff
@@ -21,8 +14,15 @@ Follow these steps to remove execution nodes from your {AAPonAWS} environment.
...
+. Generate data files by running the ansible-on-clouds-ops container.
+. Update the data file.
 . Run the `ansible-on-clouds-ops` container to remove the extension nodes.
 
-include::topics/proc-aap-aws-prepare-environment-to-remove.adoc[leveloffset=+1]
+include::topics/con-aap-aws-pull-container-image.adoc[leveloffset=+1]
+//include::topics/proc-aap-aws-prepare-environment-to-remove.adoc[leveloffset=+1]
+:context: remove
+include::topics/ref-aws-permissions-to-manage-nodes.adoc[leveloffset=+1]
+include::topics/proc-aap-aws-generate-data-files.adoc[leveloffset=+1]
```
Comment (ariordan-redhat) [inline stories/assembly-aap-aws-removing-extension-nodes.adoc:26]:
```suggestion
include::topics/proc-aap-aws-generate-remove-data-files.adoc[leveloffset=+1]
```

File: stories/assembly-aap-aws-removing-extension-nodes.adoc [hunk 2]
```diff
@@ -21,8 +14,15 @@ Follow these steps to remove execution nodes from your {AAPonAWS} environment.
...
+include::topics/proc-aap-aws-update-data-file.adoc[leveloffset=+1]
```
Comment (ariordan-redhat) [inline stories/assembly-aap-aws-removing-extension-nodes.adoc:27]:
```suggestion
include::topics/proc-aap-aws-update-remove-data-file.adoc[leveloffset=+1]
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Added suggestions to update included files - I guessed the filenames, though, so best. to check my suggestions first.

File: stories/topics/proc-aap-aws-generate-remove-data-files.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE \
+command_generator_vars aws_remove_extension_nodes \
+--output-data-file /data/extra_vars.yml
+----
+
+. When you have run these commands, a `command_generator_data/extra_vars.yml` template file is created. 
+This template file resembles the following:-
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-generate-remove-data-files.adoc:25]:
```suggestion
This template file resembles the following:
```

File: stories/topics/proc-aap-aws-removing-extension-nodes.adoc [hunk 4]
```diff
@@ -1,39 +1,30 @@
...
-$ docker pull $IMAGE
+$ docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE command_generator --data-file /data/extra_vars.yml
 ----
-
-. Run the upgrade playbook as a container using the following command.
-Replace `<removal-name>` with your foundation deployment name. 
-
-[source,bash]
++
+Provides the following command:
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-removing-extension-nodes.adoc:13]:
```suggestion
The command generator output provides the following command:
```

File: stories/topics/proc-aap-aws-update-remove-data-file.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+For example, `~/.aws/credentials`
+* `deployment_name` is the name of the foundation deployment. 
+This is the same name you used when you deployed the foundation.
+* `aws_autoscaling_group_name` is the name of the AWS AutoScaling Group to create for the extension nodes.
+* `aws_launch_template_name` is the name of the AWS EC2 launch template to create.
+* `aws_region` is the region where the foundation deployment is located.
+
+After populating the data file, it should resemble the following:
+
+The values below are provided as examples:
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-update-remove-data-file.adoc:18]:
```suggestion
The values below are provided as examples.
```

**Review by ariordan-redhat — COMMENTED** [review]
Just a few suggestions.

**Pull request merged**